scripts/updateURLs.js

```diff
@@ -0,0 +1,77 @@
+// @ts-enable
```

Comment:
wait. It's `//@ts-check`.

Reply:
Hah, that does explain why it wasn't doing it :D

Reply:
Thanks!